src/Symfony/Component/Security/Core/VoteEvents.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Security\Core;
+
+/**
+ * Contains events dispatched during access decision manager votes.
+ *
+ * @author Laurent VOULLEMIER <laurent.voullemier@gmail.com>
+ */
+final class VoteEvents
```

Comment:
I would remove this class and "harcode" the event name as we don't want people to listen to this event (which is purely internal and only available in debug mode anyway).